patchwork-registries/README.md

```diff
@@ -0,0 +1,25 @@
+# Implemented features
+1. `RegistryEvent.NewRegistry` for allowing forge mods to add their own registry
+1. `RegistryEvent.Register` for different in-game object registration and forge mods' custom registry 
+1. StructureFeature and Feature registry
+1. IForgeRegistry callbacks: CreateCallback, AddCallback and ClearCallback
+1. Default entry support
+1. Slave maps for wrapped vanilla registries: IForgeRegistry#getSlaveMap
+
+# TODO
+1. StructureFeature registration needs more testing
+1. A fully modifiable Forge mod registry implementation
+1. Code clean up in ForgeRegistries, the entry order should match Forge's version.
+1. Custom forge registries and Patchwork's wrapper implementation, see GameData:
+```
+public static final Identifier MODDIMENSIONS = new Identifier("forge:moddimensions");
+public static final Identifier SERIALIZERS = new Identifier("minecraft:dataserializers");
+public static final Identifier LOOT_MODIFIER_SERIALIZERS = new Identifier("forge:loot_modifier_serializers");
+```
+1. Make fabric registries visible to Forge mods (Carefully consider this!)
```

Comment:
I got the point, so ForgeRegistry will only wrap vanilla registries and handle forge mod registries, forge mods cannot access fabric registry via ForgeRegistry, but they can access them via the vanilla registry system. This will make our life a lot easier